src/Cms/Collection.php

```diff
@@ -79,6 +79,14 @@ public function __set(string $id, $object): void
 		$this->data[$id] = $object;
 	}
 
+	/**
+	 * Internal remover for each object in the Collection
```

Comment:
Should we add a comment on these methods here, why we're overwriting the toolkit class methods?